src/styles/base.css

```diff
@@ -61,9 +61,9 @@ main {
 }
 
 .with-x-padding > table > tbody > tr:not(:last-child) td:first-child::after {
-	@apply content-[''] w-6 block bg-theme-background h-px absolute left-0
+	@apply content-[''] w-6 block bg-theme-background h-full absolute left-0
```

Comment:
When using a tooltip on the first cell of a column, the white padding line was always being placed in the middle (might be due to the floating position of the tooltip), causing the padding not to appear. However, using the full height of the cell fixed this.